src/main/java/io/ebean/config/dbplatform/postgres/PostgresPlatform.java

```diff
@@ -42,7 +42,11 @@ public PostgresPlatform() {
     this.dbIdentity.setIdType(IdType.IDENTITY);
     this.dbIdentity.setSupportsGetGeneratedKeys(true);
     this.dbIdentity.setSupportsSequence(true);
-
+    
+    this.dbDefaultValue.setToDate("${}");
+    this.dbDefaultValue.setToTime("${}");
+    this.dbDefaultValue.setToTimestamp("${}");
```

Comment:
these are the platform specific functions.
`TIMESTAMP(${})` will convert `{ ts '2001-01-01 12:34:56' }` to `TIMESTAMP('2001-01-01 12:34:56')`